.rubocop.yml

```diff
@@ -0,0 +1,508 @@
+require: rubocop-rspec
```

Comment:
Thanks for the info.

I only have it enabled on a couple of modules and am still not sure on all of the cops that I want to enforce.

https://github.com/ghoneycutt/puppet-module-nscd